### DISCUSSION TITLE
韓國明年推「AI課本」，為什麼老師支持、家長反對？

### DISCUSSION THREAD

**Discussion by kingsophy** [1101-GS4719-C/Python-Community-GS4719-C#2063] [General]
https://www.cw.com.tw/article/5131596
韓國明年推「AI課本」，為什麼老師支持、家長反對？

隨著現今社會科技日新月異，日常生活中會用到AI的地方越來越多，這也就容易造成更多社會問題。
文中家長的擔憂不外乎是認為如果過度依賴數位設備可能會損害兒童的整體福祉和閱讀能力。事實上，國際中有些國家正趨向減少在學校中使用數位設備，強調低科技含量的學習經驗像是瑞典。
現今台灣有開始使用數位設備上課的趨勢，是否是一個利大於弊的措施仍需要時間去證明。對我而言使用數位設備上課是一把雙刃劍，它可以幫助我更好的學習新知但也容易讓我分心，造成反效果。

對各位來說使用數位設備來輔助上課是否是一個利大於弊的方法?
歡迎在下方留言分享想法~

**Comment by Tina-Peng-1566** [comment]
AI融入課程教學或許是不可避免的趨勢，但我認為，「AI課本」可能只能推行在高中、大學以上的教育階段。
畢竟如文中的家長所擔憂的，對國中以下學生來說，除了影響視力、閱讀專注度，他們也還無法控制自己的使用時間，只會更容易對3C產品上癮，這與「AI課本」想提升學習成效的初衷來說，是完全相反的。

**Comment by ds0w0** [comment]
我認為使用數位設備來輔助上課是一項值得支持的措施，因為它可以提供更客製化的學習體驗，幫助學生根據自己的學習進度和能力來獲得適當的教育。然而，這樣的技術也會加劇學生自律能力的兩極化。那些自律性強的學生能夠有效利用數位設備提升學習效率，而自律性較弱的學生則可能會受到數位設備的干擾，容易分心，甚至沉迷於娛樂性內容。因此，如何平衡數位設備的使用並確保學生能夠自律地專注於學習，是實施這項措施時需要考慮的重要課題。老師和家長在這方面的監督與指導也變得更加重要。為了避免潛在的弊端，我認為使用數位設備來輔助上課必須設有特定的課程或年齡層，且指導者必須小心管理。

**Comment by hunteryang0108** [comment]
我認為這個構想是很不錯，但是能否直接對所有人推行是需要經過審慎考慮的。教育本身就是一門深奧的學問，為了更好的教學傳承知識人類花費非常多時間研究，不慣改革推行新的方法。不過這個AI課本的出現，有點像是推翻了前人的努力，雖然可能獲得不錯的成效，但更像是在拿新生代做實驗，我個人不是很支持這麼快就實際投入使用。
